src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,191 @@
+                    }
+                    LocalDate date = LocalDate.parse(deadline[1].trim(), formatter1);
+                    commands.add(new Deadline(deadline[0], date));
+                    if (isDone == 1) {
+                        doneTasks++;
+                        commands.get(commands.size() - 1).markAsDone();
+                    }
+                    taskToAdd = true;
+                }
+                break;
+            case EVENT:
+                if (input.equals("")) {
+                    throw new EmptyTaskDescriptionException();
+                } else {
+                    String[] event = input.split("/at");
+                    if (event.length < 2) {
+                        throw new NoTimeException();
+                    }
+                    LocalDateTime time = LocalDateTime.parse(event[1].trim(), formatter2);
+                    commands.add(new Event(event[0], time));
+                    if (isDone == 1) {
+                        doneTasks++;
+                        commands.get(commands.size() - 1).markAsDone();
+                    }
+                    taskToAdd = true;
+                }
+                break;
+            case FIND:
+                if (input.equals("")) {
+                    throw new MultipleKeywordException();
+                }
+                String[] keyword = input.trim().split(" ");
+                if (keyword.length > 1) {
+                    throw new MultipleKeywordException();
+                }
+                taskList.find(keyword[0], commands);
+                break;
+            case LIST:
+                taskList.list(commands);
+                break;
+            case BYE:
+                storage.saveCommands(commands);
+                ui.exitMessaage();
+                System.exit(0);
+                break;
+            case DONE:
+                if (input.equals("")) {
+                    throw new EmptyTaskNumberException();
+                } else {
+                    taskIndex = Integer.parseInt(input.trim());
+                    done(taskIndex - 1);
+                }
+                break;
+            case DELETE:
+                if (input.equals("")) {
+                    throw new EmptyTaskNumberException();
+                } else {
+                    taskIndex = Integer.parseInt(input.trim());
+                    taskList.remove(taskIndex - 1, commands);
+                }
+                break;
+            default:
+                throw new UnknownInputException();
+            }
+            if (taskToAdd) {
+                System.out.println("I have added this task:");
+                System.out.println(commands.get(commands.size() - 1));
+                System.out.println("You now have " + commands.size() + " tasks in your list");
+            }
+        } catch (DukeException e) {
+            System.out.println(e.getMessage());
+        }
+
+
+    }
+
+    void done(int listNumber) {
```

Comment:
maybe use a method name that is easier to understand or more specific??

Reply:
the list, done and find methods were named according to their corresponding command in the chatbot